src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,37 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private double excavatorReach;
```

Comment:
Don't forget about access modifiers for your custom fields. It's better to make 'excavatorReach' private and provide a public getter to access it.